crypto/kzg/benches/benchmark.rs

```diff
@@ -0,0 +1,27 @@
+use c_kzg::KzgSettings;
+use criterion::{criterion_group, criterion_main, Criterion};
+use eth2_network_config::TRUSTED_SETUP_BYTES;
+use kzg::TrustedSetup;
+use peerdas_kzg::{PeerDASContext, TrustedSetup as PeerDASTrustedSetup};
+
+pub fn bench_init_context(c: &mut Criterion) {
+    c.bench_function(&format!("Initialize context peerdas-kzg"), |b| {
+        b.iter(|| {
+            const NUM_THREADS: usize = 1;
+            let trusted_setup = PeerDASTrustedSetup::default();
+            PeerDASContext::with_threads(&trusted_setup, NUM_THREADS)
+        })
+    });
+    c.bench_function(&format!("Initialize context c-kzg (4844)"), |b| {
+        b.iter(|| {
```

Comment:
Commenting on this, so we remember to remove it if not needed -- should be able to run `cargo bench` to check the time it takes to initialize the context